packages/grafana-ui/src/components/Combobox/Combobox.test.tsx

```diff
@@ -241,7 +241,20 @@ describe('Combobox', () => {
       await userEvent.keyboard('{Enter}');
 
       expect(screen.getByDisplayValue('Use custom value')).toBeInTheDocument();
-      expect(onChangeHandler).toHaveBeenCalledWith(expect.objectContaining({ value: 'Use custom value' }));
+      expect(onChangeHandler).toHaveBeenCalledWith(expect.objectContaining({ description: 'Use custom value' }));
+    });
+
+    it('should allow creating a custom value just when the value does not exist previously', async () => {
```

Comment:
I don't quite understand this test case, it feels like this is already covered in the one above. What you should test is that you don't find `Use custom value` if type `Option 4`.